src/os/bluestore/BlueStore.h

```diff
@@ -1319,7 +1313,9 @@ class BlueStore : public ObjectStore,
 	  get_blob().calc_offset(0, nullptr);
     }
 #endif
-    
+private:
+    void _decode(const ceph::buffer::list& v);
+    void _pin_control();
```

Comment:
should this function be deleted?

Reply:
or you meant to use this function, but i didn't see where it was called

Reply:
no, this is just a leftover from an intermediate fix. Removing...